harvard-lts/FITSservlet
# LIBDRS-9097 Add Docker build script; bump POM version

**JIRA Ticket**: [LIBDRS-9097](https://jira.huit.harvard.edu/browse/LIBDRS-9097)

# What does this Pull Request do?
Creates a Docker build script for this project. This is an interim script because it pulls from the Harvard LTS Artifactory which is not public. Changes in the future will eventually pull FITS from Maven Central.

# How should this be tested?
- From the root directory of the project run `./docker/build-fitsservlet-docker-image.sh 1.6.0-SNAPSHOT`
- Verify the Docker image was built: `docker images`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n/a
- integration tests? n/a

# Interested parties
@cvicary 

[LIBDRS-9097]: https://at-harvard-sandbox-502.atlassian.net/browse/LIBDRS-9097?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
